### PR TITLE
Implement some WotOG Druid cards

### PR DESCRIPTION
**Pull request by Epix37**

### CODE REVIEW

File: fireplace/cards/wog/druid.py [hunk 1]
```diff
@@ -4,4 +4,10 @@
 ##
 # Minions
 
+class OG_313:
+	"Addled Grizzly"
+	events = Summon(CONTROLLER, MINION - SELF).after(
```
Comment (jleclanche) [inline fireplace/cards/wog/druid.py]: Do you need `- SELF`?

File: tests/test_wog.py [hunk 2]
```diff
@@ -96,6 +96,17 @@ def test_hallazeal_the_ascended():
 	assert hallazeal.dead
 
 
+def test_mire_keeper():
+	game = prepare_game()
+	game.player1.give("OG_202").play(choose="OG_202a")
+	assert len(game.player1.field) == 2
+	game.end_turn()
+	game.end_turn()
```
Comment (jleclanche) [inline tests/test_wog.py]: Stylistic fix, whenever you want to do a full turn cycle, do `game.end_turn(); game.end_turn()`. Also make sure to follow it up with a blank line so we can quickly see the turns.

File: tests/test_wog.py [hunk 3]
```diff
@@ -109,6 +109,21 @@ def test_hallazeal_the_ascended():
...
 
+def test_mark_of_yshaarj():
+	game = prepare_game()
+	mark = game.player1.give("OG_048") 
+	mark2 = game.player1.give("OG_048") 
+	wisp = game.player1.give(WISP)
+	chicken = game.player1.give("EX1_009")
+	wisp.play()
+	chicken.play()
+	assert len(game.player1.hand) == 6
```
Comment (jleclanche) [inline tests/test_wog.py]: Whenever I need to check what's in the hand, I discard the cards from player1 at the beginning of the game (cf. other tests).

File: fireplace/cards/wog/druid.py [hunk 4]
```diff
@@ -36,3 +36,11 @@ class OG_047a:
...
 class OG_047b:
 	play = GainArmor(FRIENDLY_HERO, 8)
+
+class OG_048:
+	"Mark of Y'Shaarj"
+	play = Buff(TARGET, "OG_048e").then(
+		Find(Buff.TARGET + BEAST) & Draw(CONTROLLER)
+	)
+
+CS2_009e = buff(+2, +2)
```
Comment (jleclanche) [inline fireplace/cards/wog/druid.py]: That ID doesn't look right :)

File: fireplace/cards/wog/druid.py [hunk 5]
```diff
@@ -11,3 +11,13 @@ class OG_313:
 	)
 
 OG_313e = buff(+1, +1)
+
```
Comment (jleclanche) [inline fireplace/cards/wog/druid.py]: Always add two newlines between unrelated cards.

File: fireplace/cards/wog/druid.py [hunk 6]
```diff
@@ -44,3 +44,15 @@ class OG_048:
 	)
 
 CS2_009e = buff(+2, +2)
+
```
Comment (jleclanche) [inline fireplace/cards/wog/druid.py]: Always add two newlines between unrelated cards.

File: tests/test_wog.py [hunk 7]
```diff
@@ -161,3 +161,14 @@ def test_thistle_tea():
...
+
+
+def test_wisps_of_the_old_gods():
+	game = prepare_game()
+	game.player1.give("OG_195").play(choose="OG_195a")
+	assert len(game.player1.field) == 7
+	game.end_turn()
+	game.end_turn()
+	game.player1.give("OG_195").play(choose="OG_195b")
+	for i in range(7):
```
Comment (jleclanche) [inline tests/test_wog.py]:
```python
for wisp in game.player1.field:
    assert wisp.atk == wisp.health == 3
    assert wisp.id == WISP
```

File: tests/test_wog.py [hunk 8]
```diff
@@ -161,3 +161,14 @@ def test_thistle_tea():
...
 	assert len(game.player1.hand) == 3
 	assert game.player1.hand[0] == game.player1.hand[1] == game.player1.hand[2]
+
+
+def test_wisps_of_the_old_gods():
+	game = prepare_game()
+	game.player1.give("OG_195").play(choose="OG_195a")
+	assert len(game.player1.field) == 7
+	game.end_turn()
+	game.end_turn()
```
Comment (jleclanche) [inline tests/test_wog.py]: See earlier comment about `end_turn()`

**Comment by jleclanche** [comment]
Nice work! Just stylistic fixes and one id issue.

File: fireplace/cards/wog/druid.py (hunk 1 above)
Comment (Epix37) [inline fireplace/cards/wog/druid.py]: I just copied this from Knife Juggler - figured there would be a reason for it. All tests pass after removing it from Knife Juggler too though.

File: fireplace/cards/wog/druid.py [hunk 9]
```diff
@@ -11,3 +11,14 @@ class OG_313:
...
+
+class OG_202:
+	"Mire Keeper"
+	choose = ("OG_202a", "OG_202b")
+
+class OG_202a:
+	play = Summon(CONTROLLER, "OG_202c")
+
+class OG_202b:
```
Comment (jleclanche) [inline fireplace/cards/wog/druid.py]: Oh also, these all should go *after* Addled Grizzly (the cards are ordered by id)

**Pull request merged**